src/stylesheets/components/_form-login.scss

```
@@ -0,0 +1,75 @@
+$grid: (
+  small: 640px,
+);
+
+.form-login {
+  width: 100%;
+
+  &__body {
+    display: grid;
+    margin-top: 2.5rem;
+    row-gap: 2rem;
+    column-gap: 1.5rem;
```

Add a line break between styles and `@media` blocks:

```suggestion
    column-gap: 1.5rem;
    
```

---

Fixed in https://github.com/nimblehq/ic-nextjs-prabesh-dahal/pull/69/commits/6ab1c2610a0b1125855608a6e80b18ec4bb800f2